db/event_manager_test.go

```diff
@@ -424,7 +446,8 @@ func TestWebhookBasic(t *testing.T) {
 		em.RaiseDocumentChangeEvent(bodyBytes, docId, "", channels)
 	}
 	time.Sleep(50 * time.Millisecond)
```

Comment:
we can probably drop the sleeps now right? There are some long ones further down

```suggestion
```

Reply:
Yeah... I've clearly missed a few. Will go over and review again